src/scripts/main.js

```diff
@@ -1,3 +1,27 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+const notification = document.createElement('div');
+
+logo.addEventListener('click', () => {
+  const promise1 = new Promise((resolve) => {
+    resolve();
+  });
+
+  promise1.then(() => {
+    notification.classList = 'message';
+    notification.innerText = 'Promise was resolved!';
+    document.body.append(notification);
+  });
+});
```

Comment:
```suggestion
logo.addEventListener('click', () => {
  const promise1 = new Promise((resolve) => {
    resolve();
  });
});

 promise1.then(() => {
    notification.classList = 'message';
    notification.innerText = 'Promise was resolved!';
    document.body.append(notification);
  });
```
винеси ланцюжок з лісенера
на кожен проміс постав і catch i then для обробки результату